### PR TITLE
fix!: adjust brushes opacities

### PR DESCRIPTION
**Pull request by Darsh0307**
﻿GitHub Issue: close https://github.com/unoplatform/Uno.Themes/issues/1236

## PR Type

What kind of change does this PR introduce?

- [x] Documentation content changes

## Description

Replacing  the DisabledLow opacity with Disabled

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [] Updated the documentation as needed:
	- [x] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Review by Xiaoy312 — COMMENTED** [review]
rename commit to:
fix!: adjust brushes opacities

**Comment by Darsh0307** [comment]
@Xiaoy312  I did checked the commit message accordingly.

**Review by agneszitte — COMMENTED** [review]
Waiting on @NVLudwig's confirmation for the opacities values before we can merge this PR

File: src/library/Uno.Themes/Styles/Applications/Common/SharedColors.xaml
Comment (Xiaoy312) [inline src/library/Uno.Themes/Styles/Applications/Common/SharedColors.xaml:1]: should change the value of DisabledLowOpacity instead of referring to DisabledOpacity

Comment (Xiaoy312) [reply src/library/Uno.Themes/Styles/Applications/Common/SharedColors.xaml:1]: also if said value was changed (or with current change in this file), there is no need to update [button,fab,nav-view,pbox].xaml in the disabled-low brush will have the same value as disabled brushes

File: doc/material-colors.md
Comment (Xiaoy312) [inline doc/material-colors.md:45]:
```suggestion

| Key                | Value |
```

**Review by Xiaoy312 — CHANGES_REQUESTED** [review]

**Comment by Xiaoy312** [comment]
to be replaced by #1369

**Comment by agneszitte** [comment]
Closing this PR as the changes were done in https://github.com/unoplatform/Uno.Themes/pull/1369

**Pull request closed without merging**
